orttraining/orttraining/eager/opgen/opgen/generator.py

```diff
@@ -283,7 +283,9 @@ def _write_function_body(self, writer: writer.SourceWriter, mapped_func: MappedO
                     if i == 0:
                         writer.push_indent()
                     cpp_param = cpp_func.get_parameter(op_input)
-                    supported_types = ",".join([type for type in onnx_op.input_types[idx]])
```

Comment:
why not just add .sort on the existing line?
`supported_types = ",".join([type for type in onnx_op.input_types[idx]].sort())`
?

Reply:
because `TypeError: can only join an iterable`